ui/src/submissions/jobs/schemas/job.js

```diff
@@ -71,14 +71,18 @@ const jobSchema = object().shape({
         .label('Contact email'),
     })
   ),
-  deadline_date: date()
-    .required()
-    .test(
-      'is-valid-deadline-date',
-      'Deadline should be within next year',
-      isValidDeadlineDate
-    )
-    .label('Deadline'),
+  deadline_date: date().when('status', {
+    is: 'closed',
+    then: date(),
+    otherwise: date()
+      .required()
+      .test(
```

Comment:
This ckeck is done only on the front-end side, we should have it on the back-end too

Reply:
As discussed, we'll do them on another task (with possibly some other validation and refactoring to move validation to loaders from views) as this is supposed to be a simple bug fix.